splinterd/man/splinterd.1.md

```diff
@@ -1,81 +1,298 @@
-and the default value/behavior. If appropriate, explain
-how/when it's used, with tips, gotchas, and very short examples.
--->
+
+`--enable-biome`
+: (Optional feature) Enable the Biome subsystem, which provides
+  user management functions for Splinter applications.
+  The `--database` option is required when this flag is used.
+
+  Note: This flag is available when the "biome" feature is enabled.
 
 `-h`, `--help`
-: Prints help information
+: Prints help information.
+
+`--insecure`
+: Turns off certificate authority validation for insecure (self-signed)
+  development certificates. All peer certificates are accepted.
+
+`--no-tls`
+: Turns off TLS configuration and restricts `splinterd` to TCP (`raw`)
+  connections. This flag allows `splinterd` to start without the certificates
+  and keys that TLS requires. Without `--no-tls`, if `splinterd` cannot find the
+  certificates and keys required by TLS, it exits with an error.
+
+`-q`, `--quiet`
+: Decreases verbosity (the opposite of `-v`). When specified, only errors or
+  warnings will be displayed.
 
 `-V`, `--version`
-: Prints version information
+: Prints version information.
 
 `-v`
 : Increases verbosity (the opposite of -q). Specify multiple times for more
   output.
 
 OPTIONS
 =======
-<!--
-List of options (in alphabetical order) that describes what the option does,
-what arguments it takes, and the default value/behavior. If appropriate, explain
-how/when it's used, with tips, gotchas, and very short examples.
--->
 
-`-e`, `--example EXAMPLE`
-: Description about example options
+`--admin-timeout TIMEOUT`
+: Sets the coordinator timeout, in milliseconds, for admin service proposals.
+  (Default: 30000 ms (30 seconds).)
+
+  This setting affects consensus-related activities for pending circuit changes
+  (functions that use the two-phase commit agreement protocol in the Scabbard
+  service).
+
+`--advertised-endpoint` `ADVERTISED-ENDPOINT`
+: Specifies the public network endpoint for daemon-to-daemon communication
+  between Splinter nodes, if the network endpoint is not public. Use the format
+  `tcp://ip:port`. (Default: Same as the network endpoint; see
+  `-n`, `--network-endpoint`.)
+
+`--bind BIND-ENDPOINT`
+: Specifies the connection endpoint for the REST API. (Default: 127.0.0.1:8080.)
+
+`--ca-file CERT-FILE`
+: Specifies the path and file name for the trusted CA certificate.
+  (Default: `/etc/splinter/certs/ca_pem`.)
+
+  Do not use this option with the `--insecure` flag.
+
+`--cert-dir CERT-DIR`
+: Specifies the directory that contains the trusted CA certificates and
+  associated key files. (Default: `/etc/splinter/certs/`.)
+
+  This option overrides the `SPLINTER_CERT_DIR` environment variable, if set,
+  and any setting in the `splinterd` configuration file.
+
+`--client-cert CERT-FILE`
+: Specifies the path and file name for the client certificate, which is
+  used by `splinterd` when it is sending messages.
```

Comment:
this should specifiy "sending messages over TLS"

Reply:
Done.